doc/changelog.rst

```diff
@@ -74,6 +74,7 @@ Localization
 Admin
 ~~~~~
 
+- Define permissions for all admin modules
```

Comment:
I think you are actually defining `module level permissions for all admin modules`?